test/customized_redirect_to_test.rb

```diff
@@ -18,12 +18,12 @@ def test_redirect_index_url_after_create
     assert_redirected_to 'http://test.host/posts'
   end
 
-   def test_redirect_to_index_url_after_update
-     Post.stubs(:find).returns(mock_machine(:update_attributes => true))
-     assert !PostsController.respond_to?(:show)
-     put :update
-     assert_redirected_to 'http://test.host/posts'
-   end
+  def test_redirect_to_index_url_after_update
+    Post.stubs(:find).returns(mock_machine(:update_attributes => true))
+    assert !PostsController.respond_to?(:show)
+    put :update
+    assert_redirected_to 'http://test.host/posts'
+  end
```

Comment:
I would drop this commit, I can take care of it in the rubocop PR. But if you'd like to keep it it's ok too! :+1:

Reply:
Yes I was unsure too. I figured it was best to fix indentation just for this test case (I undid the indentation fix for another piece in this file) as I had to update this test anyway. If you feel strongly about not having this indentation fix here though I will remove.

Reply:
No, I don't feel strongly, we can update it here! :+1: